### PR TITLE
fix(curriculum): updated description of step 40 of music player

### PR DESCRIPTION
**Pull request by alt-shreya**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #53702

### CODE REVIEW

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-string-and-array-methods-by-building-a-music-player/6552303a9a78704f8ff072e9.md [hunk 1]
```diff
@@ -7,10 +7,14 @@ dashedName: step-40
 
 # --description--
 
+Now it is time to test out the pause button.
+
 You need to hook up the `pauseSong` function to an event listener to make it work.
 
 Add a `click` event listener to the `pauseButton` element, then pass in `pauseSong` as the second argument of the event listener. This is the function the event listener will run.
```
Comment (gikf) [inline curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-string-and-array-methods-by-building-a-music-player/6552303a9a78704f8ff072e9.md:14]:
```suggestion
Add a `click` event listener to the `pauseButton` element, then pass in `pauseSong` as the second argument of the event listener. This is the function the event listener will run.
```
From my understanding of the related issue, original first sentence should be removed.

Comment (alt-shreya) [reply curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-string-and-array-methods-by-building-a-music-player/6552303a9a78704f8ff072e9.md]: @gikf Thank you for the review! 

I missed it the first time, my bad 😅

**Review by gikf — APPROVED** [review]

**Review by huyenltnguyen — APPROVED** [review]
LGTM 👍

@alt-shreya Thank you for the PR and congratulations on your first contribution to this repo 🎉
We look forward to your next.

Happy contributing!

**Comment by alt-shreya** [comment]
> LGTM 👍
> 
> @alt-shreya Thank you for the PR and congratulations on your first contribution to this repo 🎉 We look forward to your next.
> 
> Happy contributing!

@huyenltnguyen Thanks for merging my PR! Can't wait to contribute more! Lesssgoooo! 🎉

**Pull request merged**